internal/operators/ocs/validations.go

```diff
@@ -228,13 +232,17 @@ func (o *operator) setStatusInsufficientResources(nodeInfo *resourceInfo, mode o
 }
 
 // count all disks of drive type ssd or hdd
-func getValidDiskCount(disks []*models.Disk) int {
-	var countDisks int
-
+func (o *operator) getValidDiskCount(disks []*models.Disk) (int64, error) {
```

Comment:
Is it required to make this a method of the operator?

Reply:
No, I will change that